src/Config.php

```diff
@@ -859,6 +859,26 @@ protected function parseConnectionParams($params, $defaults = [])
     }
 
     /**
+     * Sanity check for slashes in in taxonomy slugs.
+     */
+    public function checkTaxonomy()
+    {
+        foreach ($this->data['taxonomy'] as $key => $taxonomy) {
+            if (!empty($taxonomy['options']) && is_array($taxonomy['options'])) {
+                foreach ($taxonomy['options'] as $optionkey => $optionvalue) {
+                    if (strpos($optionkey, '/') !== false) {
+                        $error = Trans::__(
+                            'general.phrase.invalid-taxonomy-slug',
+                            ['%taxonomy%' => $key, '%option%' => $optionvalue]
+                        );
+                        $this->app['logger.flash']->error($error);
+                    }
+                }
+            }
+        }
+    }
```

Comment:
PSR-2 on the variable name case.

Also the nesting is a tad aggressive… Does this read better to you on review?
```php
    public function checkTaxonomy()
    {
        foreach ($this->data['taxonomy'] as $key => $taxonomy) {
            if (empty($taxonomy['options']) || !is_array($taxonomy['options'])) {
                continue;
            }
    
            foreach ($taxonomy['options'] as $optionKey => $optionValue) {
                if (strpos($optionKey, '/') === false) {
                    continue;
                }
    
                $error = Trans::__(
                    'general.phrase.invalid-taxonomy-slug',
                    ['%taxonomy%' => $key, '%option%' => $optionValue]
                );
                $this->app['logger.flash']->error($error);
            }
        }
    }
```